src/components/LoanCards/LoanCardImage.vue

```diff
@@ -0,0 +1,63 @@
+<!--
+	issues:
+
+	* tracking click on image - do here or bubble up?
+		kvtrackevent="Lending|click-Read more|Photo|{{id}}|true"
+
+	* do we need to deal with image editing?
+		{{#is_admin}}
+			<div class="image-edit-buttons">
+				<a class="edit" href="{{image.edit_url}}" target="_blank"><svg class="icon icon-edit"><use xlink:href="#icon-edit"/></svg></a>
+				<a class="refresh" href="#"><svg class="icon icon-refresh"><use xlink:href="#icon-refresh"/></svg></a>
+			</div>
+		{{/is_admin}}
```

Comment:
I think we should at least file a ticket for this. We can then discuss the priority further. We potentially don't have to worry about this until the logged-in-user loan card case, as you have to be logged in to be an admin.